src/main/resources/application.properties

```diff
@@ -27,8 +27,8 @@ management.metrics.enable.spring.integration=false
 
 # caching (override type in application.properties to enable)
 spring.cache.type=none
-spring.cache.cache-names=form_session
-spring.cache.caffeine.spec=maximumSize=500,expireAfterAccess=300s
+spring.cache.cache-names=form_session,case_search
+spring.cache.caffeine.spec=maximumSize=500,expireAfterAccess=300s,expireAfterWrite=300s
```

Comment:
@snopoke Is it ok / good practice for us to use the same timings and such for all of these, or should be be trying to tune per-cache?

Reply:
This is fine unless the tuning requirements are vastly different. We did discuss this briefly and decided that apply the same expiration to the session caching wouldn't be an issue. Now that I look at it again I think it would be better to have them separate to avoid expiring the session cache in the middle of a form session. I don't think it's worth blocking the PR. Once this is merged I'll implement this pattern: https://medium.com/@d.lopez.j/configuring-multiple-ttl-caches-in-spring-boot-dinamically-75f4aa6809f3